src/interface/variable.hpp

```diff
@@ -76,6 +78,11 @@ class CellVariable {
   ///< retrieve metadata for variable
   Metadata metadata() const { return m_; }
 
+  /// Get Sparse ID (-1 if not sparse)
+  int GetSparseID() const { return sparse_id_; }
+
+  bool IsSparse() const { return sparse_id_ >= 0; }
```

Comment:
Can you make these noexcept? Or just inline them?

Reply:
They should be inlined automatically, but I guess adding `inline` can't hurt.